src/main/java/it/pagopa/transactions/controllers/TransactionsController.java

```diff
@@ -53,7 +57,11 @@ public Mono<ResponseEntity<NewTransactionResponseDto>> newTransaction(
                                                                           ClientIdDto clientIdDto,
                                                                           ServerWebExchange exchange
     ) {
-        return newTransactionRequest
+        TransactionId transactionId = new TransactionId(UUID.randomUUID());
+        String jwtToken = jwtTokenUtils.generateToken(transactionId);
```

Comment:
In this way a jwt token is generated even for requests with invalid notice codes, maybe it's better to generate the jwt token after pagoPA nodo activation? where was session-service client invocation?

Reply:
Yes, you're right, I'll move the token generation in place of the session service invocation

Reply:
Modification addressed into https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/121/commits/f1619dca36997c33067c1f7b842e05e64942c277

I'he done this modifications only in this PR that take in charge the sessions-service remove and not in #120 that have the focus on JWT token generation logic porting